src/map.cpp

```diff
@@ -6554,6 +6554,25 @@ void map::loadn( const int gridx, const int gridy, const int gridz, const bool u
     abs_sub.z = old_abs_z;
 }
 
+
+template <typename Container>
+void map::smoke_meat(const tripoint &p, Container &items, const time_duration &time_since_last_actualize){
```

Comment:
Looks like you need to run astyle. I see a number of instances aside from this one of improper parentheses. Style is `f( int x )` not `f(int x)`.